### ISSUE TITLE
How can i export openshift native resource defination to yaml file, like Project/DeploymentConfig/BuildConfig

### ISSUE THREAD

**Issue by chestarss** [openshift/openshift-apiserver#215]
I want export openshift resource defination like CRD YAML file, which import to k8s api server.

from openshift:
`oc get crd project.openshift.io > project_crd.yaml`

to k8s
`kubectl apply -f project_crd.yaml`

why i want do this?
because i use karmada component for mulit cluster manager,  my member cluster is openshift. karmada control panel only support kube-apiserver, which not have Project resource.

**Comment by MichaelWasher** [comment]
There is no CRDs for the internal OpenShift APIServer types. The CR types are provided by the API aggregation step, making the OpenShift-API a required component for providing these endpoints. There are mutations and validations that are performed within the OpenShift API that will also not run if you are able to export the types as CRDs for use with KubePAI alone.

**Comment by MichaelWasher** [comment]
Or do you just want a type called Project, with no desire for the additional functionality such as ServiceAccount creation and isolation?

**Comment by chestarss** [comment]
> Or do you just want a type called Project, with no desire for the additional functionality such as ServiceAccount creation and isolation?

yes, i just want type define not implement .   I use it for karmada-apiserver which base kube-apiserver for multi-cluster management. karmada-apiserver not support proxy openshift only types

**Comment by amila77kashyapa** [comment]
Hi MichaelWasher,

I have same question like how do we export native CRDs like deploymentConfig..etc. My use case is "we are using CDK8s for our manifest generating and it is very easy to import existing  CRD into libraries but didn't find a way to get this openshift native API CRDs "deploymentconfig/route/..etc".
